lib/pi_piper/sysfs/sysfs.rb

```diff
@@ -0,0 +1,73 @@
+module PiPiper
+  class Sysfs < Driver
+
+    def initialize
+      @exported_pins = Set.new
+    end
+
+    def close
+      unexport_all
+      @exported_pins.empty? 
+    end
+    
+# Support GPIO pins
+    GPIO_HIGH = 1
+    GPIO_LOW  = 0
```

Comment:
You can move these to the top, right after the class declaration. That's usually where constants are declared.